packages/launchpad/src/test/launchpad.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import Launchpad from '../launchpad';
+import '@testing-library/jest-dom';
+import { Task } from '../types';
```

Comment:
This also looks like it should be a type import.
```suggestion
import type { Task } from '../types';
```